### DISCUSSION TITLE
Custom X-Ref Reload Notification

### DISCUSSION THREAD

**Discussion by jakedgy** [k8debug/vpk#2] [Ideas]
Can we add a notification box after a user modifies something in X-Ref Maintenance that indicates that the snapshot needs to be re-parsed before the new X-Ref rules take effect?  It seems like the most common use case is that a user adds/updates one or more X-Ref rules then reloads the same snapshot they were already viewing.

I think we could just taint the X-Ref and display a warning label to the effect of:

> This X-Ref was modified after the snapshot was parsed.  Refresh the snapshot. (links to automatically refresh the current snapshot)

**Comment by k8debug** [comment]
Thanks for the feedback.  The recommendation make sense.  Will add this request to the next update.
